### PR TITLE
Feature/bar location (get data from google api)

### PR DESCRIPTION
**Pull request by Aronsmit2**
Able to connect to google api, and search for nearby bars within a radius of 500 m.
Retrieved data pushed into an array and displayed in the terminal (for now).

### CODE REVIEW

File: app/controllers/location.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+const https = require("follow-redirects").https;
```
Comment (kriskuiper) [inline app/controllers/location.js]: Please add a link to the tutorial in Slack so we can see what you did

File: app/controllers/location.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+require("dotenv").config();
+
+const placeDetails = function() {
+    this.places = [];
+};
+
+//Declare loction information
+const googleApiKey = process.env.GOOGLE_API;
```
Comment (kriskuiper) [inline app/controllers/location.js:9]: Please add the .env property and value in slack so I can add it to Heroku

File: app/controllers/location.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+const latitude = 52.361778;
+const longitude = 4.907370;
+const radius = 500;
+
+
+//Find places within the specified radius, based on the coordinates provided.
+function placeSearch(latitude, longitude, radius) {
+    https.request({
+        host: "maps.googleapis.com",
+        path: "/maps/api/place/nearbysearch/json?location=" + latitude + "," + longitude + "&radius=" + radius + "&type=bar&key=" + googleApiKey,
```
Comment (kriskuiper) [inline app/controllers/location.js]: This should be concatenated by using ES6 backtics
```suggestion
        path: `/maps/api/place/nearbysearch/json?location=${latitude},${longitude}&radius=${radius}&type=bar&key=${googleApiKey},`
```

File: app/controllers/location.js [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+//Place retrieved data in an array, and display data in terminal (for now)
+function PlaceResult(response) { 
+    let p;
+    let data = "";
+    let sdata = "";
+    const PD = new placeDetails();
+
+    response.on("data", (chunk) => {
```
Comment (kriskuiper) [inline app/controllers/location.js]: The 'chunk' param parentheses could be omitted:
 ```suggestion
    response.on("data", chunk => {
```

File: app/controllers/location.js [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        data += chunk;
+    });
+    response.on("end", () => {
+        sdata = JSON.parse(data);
+        if (sdata.status === "OK") {
+            console.log("Status: " + sdata.status);
+            console.log("Results: " + sdata.results.length);
+            for (p = 0; p < sdata.results.length; p++) {
```
Comment (kriskuiper) [inline app/controllers/location.js]: Why don't you use `i` as the index?

**Review by kriskuiper — CHANGES_REQUESTED** [review]
See comments

File: app/controllers/location.js (hunk 2 above)
Comment (Aronsmit2) [reply app/controllers/location.js:9]: gedaan, zie slack 😄

File: app/controllers/location.js (hunk 3 above)
Comment (Aronsmit2) [reply app/controllers/location.js]: is aangepast

File: app/controllers/location.js (hunk 4 above)
Comment (Aronsmit2) [reply app/controllers/location.js]: ook aangepast

File: app/controllers/location.js (hunk 5 above)
Comment (Aronsmit2) [reply app/controllers/location.js]: ah i don't know, changed back to I as index

**Review by kriskuiper — APPROVED** [review]
Approved

**Pull request merged**